platforms/chibios/converters/promicro_to_elite_pi/_pin_defs.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2022 QMK
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+
+// Left side (front)
+#define D3 0U
+#define D2 1U
+//      GND
+//      GND
+#define D1 2U
+#define D0 3U
+#define D4 4U
+#define C6 5U
+#define D7 6U
+#define E6 7U
+#define B4 8U
+#define B5 9U
+
+// Right side (front)
+//      RAW
+//      GND
+//      RESET
+//      VCC
+#define F4 29U
+#define F5 28U
+#define F6 27U
+#define F7 26U
+#define B1 22U
+#define B3 20U
+#define B2 23U
+#define B6 21U
```

Comment:
Sorry, missed this off the initial suggestions,

If there is a user led, then update to whatever
```suggestion
#define B6 21U

// LEDs
#define D5 12U
#define B0 13U
```

Reply:
No LEDs on this controller

Reply:
Doesnt matter, they should be still mapped to "something" so that pro micro code still compiles.